modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordActiveConfigWatch.kt

```diff
@@ -1,56 +1,64 @@
 package com.metalbear.mirrord
 
 import com.intellij.notification.NotificationType
+import com.intellij.openapi.application.ReadAction
 import com.intellij.openapi.vfs.AsyncFileListener
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.openapi.vfs.VirtualFileManager
 import com.intellij.openapi.vfs.newvfs.events.VFileDeleteEvent
 import com.intellij.openapi.vfs.newvfs.events.VFileEvent
 import com.intellij.openapi.vfs.newvfs.events.VFileMoveEvent
 
 class MirrordActiveConfigWatch(private val service: MirrordProjectService) : AsyncFileListener {
+    /**
+     * @param url null means that the file has been removed
+     */
+    class ConfigMovedTo(val url: String?)
+
     override fun prepareChange(events: MutableList<out VFileEvent>): AsyncFileListener.ChangeApplier? {
-        val notification = service
+        val result: ConfigMovedTo? = service
             .activeConfig
             ?.url
             ?.let { activeUrl ->
                 events
-                    .filter { it.file?.url == activeUrl }
+                    .filter { it.file?.let { file -> activeUrl.startsWith(file.url) } ?: false }
```

Comment:
Why is this startsWith rather than ==?

Reply:
Also is the ?: false necessary? If the clause returns null will the filter include the item?

Reply:
`startsWith` allows to also find changes in parent directories (e.g. config at `.mirrord/mirrord.json` and someone moves `.mirrord` to `.mirrorddd`)

The `?: false` is necessary, because the first part of the expression resolves to a `null` when the event does not contain any file (I thought: weird but ok)